src/database.ts

```diff
@@ -1254,23 +1455,26 @@ class Database extends ServiceObject {
    * @param {RunUpdateCallback} [callback] Callback function.
    * @returns {Promise<RunUpdateResponse>}
    */
-  runPartitionedUpdate(query, callback) {
+  runPartitionedUpdate(
+    query: string | ExecuteSqlRequest,
+    callback: RunUpdateCallback
+  ): void | Promise<[]> {
```

Comment:
The Promise response doesn't look right, also looks like we're missing overloads for this method.